content/release-10-16-0/applications-10-16-0-bundle/10-16-0-0.md

```diff
@@ -113,6 +122,18 @@ Cockpit</td>
 MTM-47676</td>
 </tr>
 
+<tr>
+<td>Cockpit</td>
+<td>An issue has been fixed which lead to showing an error message for the Welcome widget for older tenants using a deprecated version of the Home dashboard.</td>

```

Comment:
```suggestion
<td>An issue has been fixed which led to showing an error message for the Welcome widget for older tenants using a deprecated version of the Home dashboard.</td>
```

maybe even "An issue has been fixed which showed/displayed...", but depends on the style choice.